userdata/testAPI.py

```diff
@@ -0,0 +1,18 @@
+'''
+Created on Mar 16, 2015
+
+@author: raju332
+'''
+import VeloAPI

```

Comment:
Is it getting run? I didn't see any mention of it in the travis log. If it isn't, can we package it up in a Nose test or something? If it's not getting run, it's not doing any good.